pkg/mapper/configmap/configmap.go

```diff
@@ -146,6 +152,11 @@ func ParseMap(m map[string]string) (userMappings []config.UserMapping, roleMappi
 	return userMappings, roleMappings, awsAccounts, err
 }
 
+func isSkippable(data string) bool {
+	trimmed := strings.TrimSpace(data)
+	return trimmed == "" || trimmed == "``" || trimmed == "\"\""
```

Comment:
What about single quotes?  Your example has one with single quotes in the PR description.